From petkan@nucleusys.com Mon Oct  3 07:42:42 2005
Date: Mon, 3 Oct 2005 17:24:11 +0300 (EEST)
From: Petko Manolov <petkan@nucleusys.com>
To: Greg KH <greg@kroah.com>
Subject: [PATCH] pegasus.h
Message-ID: <Pine.LNX.4.63.0510031722280.2514@bender.nucleusys.com>


This one is a tiny patch adding one more device to the list.  Please 
apply. :)


Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

 drivers/usb/net/pegasus.h |    2 ++
 1 file changed, 2 insertions(+)

---
--- gregkh-2.6.orig/drivers/usb/net/pegasus.h
+++ gregkh-2.6/drivers/usb/net/pegasus.h
@@ -181,6 +181,8 @@ PEGASUS_DEV( "Accton USB 10/100 Ethernet
 		DEFAULT_GPIO_RESET )
 PEGASUS_DEV( "SpeedStream USB 10/100 Ethernet", VENDOR_ACCTON, 0x5046,
 		DEFAULT_GPIO_RESET | PEGASUS_II )
+PEGASUS_DEV( "Philips USB 10/100 Ethernet", VENDOR_ACCTON, 0xb004,
+		DEFAULT_GPIO_RESET | PEGASUS_II )
 PEGASUS_DEV( "ADMtek ADM8511 \"Pegasus II\" USB Ethernet",
 		VENDOR_ADMTEK, 0x8511,
 		DEFAULT_GPIO_RESET | PEGASUS_II | HAS_HOME_PNA )
